src/gwt/src/org/rstudio/studio/client/common/debugging/ui/ConsoleError.ui.xml

```diff
@@ -74,8 +74,8 @@
                   ui:field="showTracebackImage"
                   width="9"
                   height="9"
-                  styleName="{style.errorCommandImage}"/>
-           <g:Anchor styleName="{style.errorCommandText}" ui:field="showTracebackText">
+                  styleName="{style.errorCommandImage} show_traceback_image"/>
```

Comment:
Can you clarify why we need these extra class names here? Are we unable to query for the class name generated by GWT in the associated `style` object?

Reply:
I just double checked and unfortunately GWT is assigning the same class to the bindings, both tex telements "Show Traceback" and "Rerun" have the class `org-rstudio-studio-client-common-debugging-ui-ConsoleError_ConsoleErrorUiBinderImpl_GenCss_style-errorCommandText`.